docs/installation/local-installation.md

```diff
@@ -0,0 +1,76 @@
+# Local Installation
+
+These guides show how to use prepared Docker Compose configuration to simplify the installation process.
+We recommend to choose **installation using Docker** because it is the easiest and fastest way to start using Shopsys Framework.
+Docker contains complete development environment necessary for running your application so you do not need to install and configure the whole server stack (Nginx, PostgreSQL, etc.) in order to run and develop Shopsys Framework on your machine.
+
+## How it works
+
+All the services needed by Shopsys Framework like Nginx or PostgreSQL are run in Docker.
+Your source code is automatically synchronized between your local machine and Docker container in both ways.
+
+That means that you can normally use your IDE to edit the code while it is running inside a Docker container.
+
+## Installation using Docker
+
+- [Linux](installation-using-docker-linux.md)
+- [MacOS](installation-using-docker-macos.md)
+- [Windows 10 Pro and higher](installation-using-docker-windows-10-pro-higher.md)
+
+## Installation without Docker
+
+If your system is not listed above or you do not want to use Docker containers, do not worry, you can still install, develop and run Shopsys Framework natively by reading the following section.
+
+First it is truly essential to read and understand the articles about requirements and configurations for Shopsys Framework application.
+1. [Application Requirements](application-requirements.md)
+1. [Application Configuration](application-configuration.md)
+1. [Installation Troubleshooting](installation-troubleshooting.md)
+
+After you read the articles you are ready to start with the creating and building the Shopsys Framework project.
+
+### Create new project from Shopsys Framework sources
+
+```
+COMPOSER_MEMORY_LIMIT=-1 composer create-project shopsys/project-base --keep-vcs
```

Comment:
this command does not work in windows power shell because of the COMPOSER_MEMORY_LIMIT